server/controllers/ssotRetrievalController.js

```diff
@@ -55,16 +55,17 @@ exports.getBotList = async (req, res) => {
     }
 };
 
-// GET /renameBot?id=Browser&newName=Open+Browser
+// GET /renameBot?id=78d09f66d2ed466cf20b06f7&newName=Bot+Browser
 exports.renameBot = async (req, res) => {
     try {
         res.set('Content-Type', 'application/json');
         const {id} = req.query;
+        const usableUserId = mongoose.Types.ObjectId(id);
```

Comment:
what is `usableuserId`? Why is it usable?

Reply:
Because MongoDB Ids are not Strings, but rather objects, so to be able to use the passed Id, it needs to be converted to such object